application-analytics-ui/pom.xml

```diff
@@ -53,7 +53,7 @@
     <!-- The list of documents that are not covered by the extension license. The user that installs the extension has
     full access to these documents (view, edit, delete) without having a license, so use this property wisely. -->
     <xwiki.extension.licensing.excludedDocuments>
-      Analytics.DefaultDashboard,
+      Analytics.WebPreferences,
```

Comment:
Shouldn't ``Analytics.Code.WebPreferences`` also be listed?

Reply:
I'm not sure if I should permit the user to edit the preferences on the Analytics.Code page because, right now, I have added viewing permissions for all groups inside Analytics.Code.WebPreferences for the tracking code to work, and if a user can modify the permissions, the tracking would stop working.

Reply:
I removed the preferences for the code space and added the viewing right on the individual pages